### DISCUSSION TITLE
How can I change hosting location for my site?

### DISCUSSION THREAD

**Discussion by Daniumy** [vercel/community#6097] [Help]
### Summary

Hello, I ve changed the functions regions from USA to France, as my site is going to be only visited by spanish people. But hosting locations checkers like https://hostingchecker.com/ still say it is located in the USA, I might have confused function regions or whatever that s called.

Is there a way to change the location of my hosting? Thanks

### Example

webllope.es

### Steps to Reproduce

https://hostingchecker.com/

**Comment by versecafe** [comment]
@Daniumy This may be because it's hitting Vercel's edge cache which will hit the nearest point to the hostingchecker meaning if the checker is hosted in the USA it will likely flag as being hosted in the US
